mdpow/fep.py

```diff
@@ -238,22 +253,33 @@ def load(cfg, section):
 
         cfg_get = {float: cfg.getfloat,
                    int: cfg.getint,
-                   str: cfg.getstr,    # literal strings, no conversion of None (which we need for the MDP!)
+                   str: cfg.getstr,  # literal strings, no conversion of None (which we need for the MDP!)
                    list: cfg.getarray  # numpy float array from list
                    }
+
         def getter(type, section, key):
             try:
                 return cfg_get[type](section, key)
-            except ConfigParser.NoOptionError:
+            except NoOptionError:
                 return None
-        return FEPschedule((key, getter(keytype, section, key)) for key,keytype in keys.items()
+
+        return FEPschedule((key, getter(keytype, section, key)) for key, keytype in keys.items()
                            if getter(keytype, section, key) is not None)
 
     def __deepcopy__(self, memo):
-        x = type(self)()
-        for k,v in self.iteritems():
-            x[k] = copy.deepcopy(v, memo)
-        return x
+        if sys.version_info.major == 3:
```

Comment:
iteritems() method not available in python 3, Iterable not variable in python 2, would like more elegant solution, but this should function.